.ci/spec/run-test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+csource="${BASH_SOURCE[0]}"
+while [ -h "$csource" ] ; do csource="$(readlink "$csource")"; done
+root="$( cd -P "$( dirname "$csource" )/../" && pwd )"
+
+. "${root}/.ci/spec/env.sh"
+. "${root}/.ci/common.sh"
+
+pushd "${root}" > "${output}"
+
+# Assumes packet is already set up
+
+# Define a custom cleanup function to destroy any orphan guests
+# on the packet device
+function cleanup() {
+    (>&2 echo "Cleaning up packet device")
+    unset PACKET_EXEC_PERSIST
+    pkt_wrap_stream "cd vagrant;vagrant destroy -f" \
+                "Vagrant command failed"
+}
```

Comment:
We don't need this cleanup function in the tests script. We can just clean everything at once when we are done.